src/sales_rankings.js

```diff
@@ -0,0 +1,50 @@
+/* Constants for Amazon Category Sales Rankings
+  SalesRankings represent the rank number a product must under to be within the top 50%
+  of item in that category. The numbers may not be exactly correct, but should be in the ballpark.
```

Comment:
Great job to get a start on this. I looked at the chart you referenced and saw you had to do some work to get all the category data.
This approach will not work long term though because the list you referenced is not dated even though rankings always change. So, we will be stuck on using an old criteria and might be eliminating items that have become popular. I have thought of a few ideas here, maybe find a list/website that updates regularly, scrape or curl it and use it here (this might be done as a pre-processing step). We can talk more about this too because it is more work than a basic heuristic